home/rvagg/2021-02-exploration-report-small-filecoin-deals.md

```diff
@@ -0,0 +1,134 @@
+    * Requesting deal conditions (price per GiB per epoch (30 seconds), min size and max size)
+    * Proposing a specific deal
+    * Having a deal accepted by a miner
+    * Transferring the data to the miner
+    * The miner including the deal data into a sector they are committing (currently or queued)
+    * Having deal confirmation placed on the chain pointing to its location in the sealed sector
+* Deals can also be made "offline" where the deal is made but the data is shipped (perhaps manually) to the miner who loads it into a sector
+* Deal funds pay out to the miner over the course of the lifetime of the deal to ensure they continue proving retention of data
+* Deals can either be regular deals or "verified deals"
+    * Verified deals come from an additional social layer outside of Filecoin, called Filecoin Plus (FIL+)
+    * Notaries are organisations that can hand out "DataCap" to clients
+    * Notaries are selected by the Filecoin community, distributed around the world
+    * "DataCap" is a quota that can be spent, along with FIL, to make verified deals
+    * Verified deals contribute 10x to miner power, proportional to how much space they occupy in a sector
+    * Therefore, verified deals should be attractive to miners who may charge less FIL to get them
+
+### Deal incentives & challenges
+
+* Miners bear responsibility for publishing deals to the chain, with high gas costs this can be in the order of $5
+* Recent changes allow for batch publishing deals, but anecdotally this hasn't changed the economics for miners much (need more context here - is it because the batch publish is still quite expensive, or because there are not enough deals to batch?)
+* Accepting deals can offset the cost of sealing and proving, but this biases toward larger deals
+* Verified deals have the 10x increase in power, but they are scarce because FIL+ is so new, currently unknown the scale of incentive impact for miners for large and small deals
+* Bugs in Lotus have made miners gun-shy on doing anything more than CC:
+  * Transferring deal data is buggy and transfers often fail (anecdotal)
+  * Retrieval is buggy and can cause crashes, which are problematic when you need 100% uptime to prove sectors (anecdotal)
+* A lot of large miners don't seem to have transitioned from CC sealing to accepting deals - could be immaturity of market
+* Many miners don't respond to `query-ask` at all. Of those that do, the majority appear to reject deals even when those deals fit within the ask parameters. So the `query-ask` mechanism seems of limited use in its current form.
+
+Filecoin currently only stores ~5 Pb of deal data and growth rate is not large.
+
+## Installing and Running Lotus
+
+Followed instructions at https://docs.filecoin.io/get-started/lotus/installation/ for an AWS instance
+
+ * Large AMD EPYC instance, Ubuntu 20.04
+ * Installed dependencies (used Go Snap rather than Go tarball)
+ * Cloned and build Lotus with CPU SHA support
```

Comment:
You actually do not need this ( nor an AMD ) for just validating the chain. The SHA-NI helps exclusively with mining.